EvidenceApi/V1/UseCase/CreateDocumentSubmissionUseCase.cs

```diff
@@ -12,10 +12,12 @@ namespace EvidenceApi.V1.UseCase
     public class CreateDocumentSubmissionUseCase : ICreateDocumentSubmissionUseCase
     {
         private readonly IEvidenceGateway _evidenceGateway;
+        private readonly IDocumentsApiGateway _documentsApiGateway;
 
-        public CreateDocumentSubmissionUseCase(IEvidenceGateway evidenceGateway)
+        public CreateDocumentSubmissionUseCase(IEvidenceGateway evidenceGateway, IDocumentsApiGateway documentsApiGateway)
         {
             _evidenceGateway = evidenceGateway;
+            _documentsApiGateway = documentsApiGateway;
         }
 
         public DocumentSubmissionResponse Execute(Guid evidenceRequestId, DocumentSubmissionRequest request)
```

Comment:
This method is kinda long. You could refactor it to pull some of the related things out into private methods and call them from here.